randovania/data/json_data/prime3.txt

```diff
@@ -2271,6 +2735,8 @@ Asset id: 14742980547553882067
       Boost Ball
 
 > Floating Island; Heals? False
+  > Pickup (Energy Tank)
+      Trivial
   > Room Center
       Boost Ball
```

Comment:
This node isn't necessary since the etank is that only relevant thing it there, so the etank can directly contact to the main area and vice versa. Also, do you need boost to return or is the cannon already active?

Reply:
I cleaned this up. Cannon on the way back is already active.